content/_data/changelogs/lts.yml

```
@@ -12944,6 +12944,20 @@
     message: |-
       Prevent Java null pointer exception when adapting a keyboard shortcut based on the User-Agent header.
 
+- version: "2.504.2"
+  date: 2025-05-28
+  changes:
+  - type: rfe
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)